apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -231,6 +231,9 @@ function initializeBlocklyWrapper(blocklyInstance) {
     },
     getBlockFields: function(block) {
       return block.getTitles();
+    },
+    isWorkspaceReadOnly: function(workspace) {
```

Comment:
I'm not quite sure I understand why we call the `workspace.isReadOnly` here, but delete it from cdoWorkspaceSVG?

Reply:
`cdoWorkspaceSvg` is extending the Google Blockly class, and is only used by labs where the blockly version is "Google".  When we call `workspace.isReadOnly` here, we're referring to a function in the cdo blockly fork. This wrapper is only used by labs where the blockly version is "cdo". Does that make sense?

Reply:
Yup, that helped straighten out my brain, thanks!